tests/templates/test_layers.py

```diff
@@ -237,55 +240,6 @@ def test_strong_ent_layers_uses_correct_number_of_imprimitives(self, n_layers, n
         types = [type(q) for q in rec.queue]
         assert types.count(imprimitive) == n_subsystems * n_layers
 
-    @pytest.mark.parametrize("n_wires, n_layers, ranges", [(2, 2, [2, 1]), (3, 1, [5])])
-    def test_strong_ent_layers_ranges_equals_wires_exception(self, n_layers, n_wires, ranges):
-        """Test that StronglyEntanglingLayers throws and exception if a range is equal to or
-        larger than the number of wires."""
-        dev = qml.device("default.qubit", wires=n_wires)
-        weights = np.random.randn(n_layers, n_wires, 3)
-
-        def circuit(weights):
-            StronglyEntanglingLayers(weights=weights, wires=range(n_wires), ranges=ranges)
-            return qml.expval(qml.PauliZ(0))
-
-        qnode = qml.QNode(circuit, dev)
-
-        with pytest.raises(ValueError, match="the range for all layers needs to be smaller than"):
-            qnode(weights)
-
-    def test_strong_ent_layers_illegal_ranges_exception(self):
-        """Test that StronglyEntanglingLayers throws and exception if ``ranges`` parameter of illegal type."""
-        n_wires = 2
-        n_layers = 2
-        dev = qml.device("default.qubit", wires=n_wires)
-        weights = np.random.randn(n_layers, n_wires, 3)
-
-        def circuit(weights):
-            StronglyEntanglingLayers(weights=weights, wires=range(n_wires), ranges=["a", "a"])
-            return qml.expval(qml.PauliZ(0))
-
-        qnode = qml.QNode(circuit, dev)
-
-        with pytest.raises(ValueError, match="'ranges' must be a list of integers"):
-            qnode(weights)
-
-    @pytest.mark.parametrize("n_layers, ranges", [(2, [1, 2, 4]), (5, [2])])
-    def test_strong_ent_layers_wrong_size_ranges_exception(self, n_layers, ranges):
-        """Test that StronglyEntanglingLayers throws and exception if ``ranges`` parameter
-        not of shape (len(wires),)."""
-        n_wires = 5
-        dev = qml.device("default.qubit", wires=n_wires)
-        weights = np.random.randn(n_layers, n_wires, 3)
-
-        def circuit(weights):
-            StronglyEntanglingLayers(weights=weights, wires=range(n_wires), ranges=ranges)
-            return qml.expval(qml.PauliZ(0))
-
-        qnode = qml.QNode(circuit, dev)
-
-        with pytest.raises(ValueError, match="'ranges' must be of shape"):
-            qnode(weights)
-
```

Comment:
I tried not to delete many tests, but our new design principle of not testing each and every input will make some tests obsolete. When I foresaw this to happen, and the test would need a rewrite, I just delete them.